interface.go

```diff
@@ -1,13 +1,20 @@
 package gorm
 
-import "database/sql"
+import (
+	"context"
+	"database/sql"
+)
 
 // SQLCommon is the minimal database connection functionality gorm requires.  Implemented by *sql.DB.
 type SQLCommon interface {
 	Exec(query string, args ...interface{}) (sql.Result, error)
```

Comment:
Just as a sanity check, do all the tests pass if you remove the non-context functions from this interface?

Reply:
Functions such as `HasIndex` calls the non-context functions so the tests will fails. Would you prefer if I change all non-context function calls with context function call with `context.Background()`?

Reply:
though that feels little unnecessary if you're going for the minimal amount of change needed for making the jaeger spans appear.

Reply:
Oh yeah good point, no need to do it then. I was just thinking if that would a low cost way of double checking that we didn't leave anything out.